src/mame/mr/dribling.cpp

```diff
@@ -246,46 +269,56 @@ void dribling_state::misc_w(uint8_t data)
 		m_maincpu->set_input_line(0, CLEAR_LINE);
 
 	// bit 6 = parata (save by goalkeeper)
-
+	// TODO: this sound is very slow, needs to be fixed.
+	// m_i_parata->write_line(BIT(data, 6));
 	// bit 5 = ab. campo (field enable)
 	m_abca = (data >> 5) & 1;
-
 	// bit 4 = ab. a.b.f.
+	m_i_enable->write_line(BIT(data, 4));
+
+
+
+
 	// bit 3 = n/c
 
 	// bit 2 = (9) = PC2
 	// bit 1 = (10) = PC1
 	// bit 0 = (32) = PC0
 	m_input_mux = data & 7;
-	LOGMISC("%s:misc_w(%02X)\n", machine().describe_context(), data);
-}
 
+}
 
 void dribling_state::sound_w(uint8_t data)
```

Comment:
Grouping is all messed up – there was one blank line between blocks in the function, and two blank lines between functions.

Reply:
Fixed